pmd-visualforce/etc/grammar/VfParser.jjt

```diff
@@ -70,7 +70,7 @@ PARSER_END(VfParser)
 | <#XMLNAME: (<ALPHA_CHAR> | "_" | ":") (<IDENTIFIER_CHAR>)* >
 | <#QUOTED_STRING_NO_BREAKS: ( "'" ( ~["'", "\r", "\n"] )* "'" )
                     | ( "\"" ( ~["\"", "\r", "\n"] )* "\"" ) >
-| <#QUOTED_STRING: ( "'" ( ~["'"] )* "'" ) | ( "\"" ( ~["\""] )* "\"" ) >
+| <#QUOTED_STRING: ( "'" ( ~["'"] )* "'" ) | ( "\"" ( ~["\""] | "\\\"" )* "\"" ) >
```

Comment:
shouldn't you change `( "'" ( ~["'"] )* "'" )` to `( "'" ( ~["'"] | "\\'" )* "'" )` to allow strings with single quotes including escaped quotes?